modules/olmv1-adding-a-catalog.adoc

```diff
@@ -73,11 +71,11 @@ $ oc describe clustercatalog
 .Example output
```

Comment:
NOTE:
Before 4.18 GA, I am going to do a sweep of the example output sections for the entire extensions book.

Reply:
yes. need to refresh the output based on the latest 4.18.

Reply:
Here is the ticket to track the work: https://issues.redhat.com/browse/OSDOCS-13342

Reply:
ok, thanks